rooms-quickstart/rooms.py

```diff
@@ -115,11 +112,11 @@ def remove_participants_from_room(self, room_id:str, participants_list:list):
     rooms.setup()
     rooms.create_room()
     rooms.update_room(room_id=rooms.roomsCollection[0])
-    rooms.update_participants(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant1, rooms.participant3])
-    rooms.get_room_after_participants_update(room_id=rooms.roomsCollection[0])
-    rooms.add_participants_to_room(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4, rooms.participant5, rooms.participant6])
+    rooms.get_room(room_id=rooms.roomsCollection[0])
+    rooms.add_or_update_participants(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4.raw_id, rooms.participant5.raw_id])
+    rooms.list_all_rooms()
     rooms.get_participants_in_room(room_id=rooms.roomsCollection[0])
-    rooms.remove_participants_from_room(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4, rooms.participant5])
+    rooms.remove_participants_from_room(room_id=rooms.roomsCollection[0], participants_list=[rooms.participant4.raw_id, rooms.participant5.raw_id])
     rooms.get_participants_in_room(room_id=rooms.roomsCollection[0])
```

Comment:
Do we need another get_participants_in_room() at the end?

Reply:
I think this is just to confirm that the remove participants worked correctly. We should no longer see the participants removed in the get_participants result list.